src/index.html

```diff
@@ -1,14 +1,79 @@
-<!DOCTYPE html>
-<html lang="en">
+
+<!doctype html>
+<html lang="en" class="h-100">
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="Mark Otto, Jacob Thornton, and Bootstrap contributors">
```

Comment:
Who are these?

Reply:
Those are the bootstrap developers. probably delete that (or change it to `IIT IPRO Students`